awx/ui/client/lib/components/input/textarea.partial.html

```diff
@@ -1,17 +1,40 @@
 <div class="col-sm-{{::col}} at-InputContainer">
     <div class="form-group at-u-flat">
         <at-input-label></at-input-label>
-
-        <textarea class="form-control at-Input at-InputTextarea"
-                  ng-model="state._value"
-                  ng-class="{ 'at-Input--rejected': state._rejected }"
-                  ng-attr-rows="{{::state._rows || 6 }}"
-                  ng-attr-maxlength="{{ state.max_length || undefined }}"
-                  ng-attr-tabindex="{{ tab || undefined }}"
-                  ng-attr-placeholder="{{::state._placeholder || undefined }}"
-                  ng-change="vm.check()"
-                  ng-disabled="state._disabled || form.disabled" /></textarea>
-
+        <div ng-class="{ 'input-group': state.tagMode }">
```

Comment:
this to me reads as if it was a component for textarea...the component seems to have shifted to be more like our lookup (with a button to the left...just with a text area instead of an input).  It is confusing for this be named textarea, based on what I understand.  Are there other forms that are using the textarea component?  (doubt it, but just a note it would be good to check).

Reply:
I see what you mean. I added this whole div to the textarea components for a very specific state where we need make the field _look_ like a tag inside of a text area:

<img width="1118" alt="Screen Shot 2019-03-14 at 12 19 30 PM" src="https://user-images.githubusercontent.com/9753817/54373337-74f5b380-4653-11e9-89af-07e17cc04428.png">

I didn't find a way of doing this by nesting a tag component inside of the actual `textarea` element, so when we need to show the component in "tag mode", we're actually just hiding the entire textarea and showing this div instead.

There are no other forms using this component. I'm open to any specific suggestions you might have here - are you proposing that we rename the component to make it clear that it is customized specifically for the credentials form?

Reply:
We talked about this offline. The improvement @jlmitch5  suggested that I'm going to make here is to update the hint text when the component is in tag mode so that it doesn't suggest that you can drag and drop or enter text (because you can't). Instead, it will give a different hint to the user that they must first unlink the credential.

Reply:
@jlmitch5 I've updated the textarea components so that they don't show the drag-and-drop hint when the input value is being displayed as a tag.